analytics/views.py

```diff
@@ -1140,12 +1141,17 @@ def support(request: HttpRequest) -> HttpResponse:
                                          event_time=event_time)
         elif request.POST.get("status", None) is not None:
             status = request.POST.get("status")
+            event_time = timezone_now()
             if status == "active":
                 do_send_realm_reactivation_email(realm)
                 context["message"] = f"Realm reactivation email sent to admins of {realm.string_id}."
+                RealmAuditLog.objects.create(realm=realm, acting_user=request.user, event_type=RealmAuditLog.REALM_REACTIVATED,
+                                             event_time=event_time)
```

Comment:
Move this to `do_send_realm_reactivation_email` and make the function take `acting_user` as an optional argument. Also `RealmAuditLog.REALM_REACTIVATED` is incorrect. It should be something like `RealmAuditLog.REALM_REACTIVATION_EMAIL_SENT`